### ISSUE TITLE
 🐛 my chat should be come bottom 

### ISSUE THREAD

**Issue by biplavmz** [narayan954/dummygram#980]
### Description

now after click on send button my text chat is going at top position in sequence but the flow should be it come at bottom.

I'm a GSSoC'23 contributor pls assign me this task under gssoc label and level 

### Steps to reproduce

go to website and click on any post commet

### Screenshots

<img width="436" alt="image" src="https://github.com/narayan954/dummygram/assets/68702055/d312b053-1985-4b69-9bce-e73ecfc29c1d">

<img width="520" alt="image" src="https://github.com/narayan954/dummygram/assets/68702055/09be26b7-af0d-4a14-952e-bcc1847ce008">

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I have checked the existing issues
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

**Comment by narayan954** [comment]
yes, just change the order at which they come :) and try to add scroll to bottom,once comment is added
